apps/studio/components/interfaces/Organization/BillingSettings/TaxID/TaxID.tsx

```diff
@@ -158,98 +149,93 @@ const TaxID = () => {
             )}
 
             {isError && (
-              <AlertError error={error} subject="Failed to retrieve organization tax IDs" />
+              <AlertError error={error} subject="Failed to retrieve organization tax Id" />
             )}
 
             {isSuccess && (
-              <Form validateOnBlur id={formId} initialValues={{}} onSubmit={onSaveTaxIds}>
-                {() => (
-                  <FormPanel
-                    footer={
-                      <div className="flex py-4 px-8">
-                        <FormActions
-                          form={formId}
-                          isSubmitting={isUpdating}
-                          hasChanges={hasChanges}
-                          handleReset={() => onResetTaxIds()}
-                          helper={
-                            !canUpdateTaxIds
-                              ? "You need additional permissions to manage this organization's tax IDs"
-                              : undefined
-                          }
-                        />
-                      </div>
+              <Panel
+                className="mb-8"
+                footer={
+                  <FormActions
+                    form="tax-id-form"
+                    isSubmitting={isUpdating}
+                    hasChanges={hasChanges}
+                    handleReset={() => onResetTaxIds()}
+                    helper={
+                      !canUpdateTaxId
+                        ? "You need additional permissions to manage this organization's tax ID"
+                        : undefined
                     }
+                  />
+                }
+              >
+                <Form_Shadcn_ {...form}>
+                  <form
```

Comment:
These could be outside of `<Panel/>` so that that _submit_ `<Button />` in `<FormActions />` can then be inside the form, and onSubmit on form can handle the submission. but no biggie.